src/main/resources/helpTemplates/common.html

```diff
@@ -0,0 +1,53 @@
+<#--
+        This file contains part of the theming used to present Barclay docs on a website. Styling is separated
+        out, so pages will be minimalistic html unless replacement styling is provided.
+        -->
+
+    <#macro footerInfo>
+        <hr>
+        <p><a href='#top'><i class='fa fa-chevron-up'></i> Return to top</a></p>
```

Comment:
@vdauwera Once thing I tried to do is to keep Barclay so that it could generate some minimally useful doc by default with minimal modifications to the template. I tried to remove anything that was truly GATK specific, but I don't mind leaving these styles in here since they make the output nicer.